File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/gettingstarted/fibonacci.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    int n;
    cin >> n;
    int a,b,c;
    a=0;
    b=1;
    cout<<a<<" "<<b<<" ";
    //write your code here
    for(int i=2; i<n; i++){
        c=a+b;
        a=b;
        b=c;
        cout<<c<<" ";
    }
    return 0;
}

File: Android/divyaagarwal24_DivyaAgarwal_2024it1100_2/GettingStarted/fibonacci.java
import java.util.*;
  
  public class fibonacci{
  
  public static void main(String[] args) {
    Scanner scn=new Scanner(System.in);
    int a=0,b=1,c,i,n;
    n=scn.nextInt();

    if(n>1)
    {
    System.out.println(a);
    System.out.println(b);
    n=n-2;
    }
    while(n!=0)
    {
       c=a+b;
       a=b;
       b=c;
       System.out.println(c);
       n--;
    }
    // write your code here
   }
  }
